src/components/Elements/Map/leaflet.ts

```diff
@@ -122,14 +123,28 @@ true;
 // inject a marker into the map to show the highlighted room
 export const _injectMarker = (
     mapRef: React.RefObject<WebView>,
-    coordinates: number[]
+    coordinates: number[],
+    colors: Colors
 ): void => {
+    const color = colors.primary
+    const svg = `
+    <svg width="400px" height="400px" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg"><path d="M512 85.3c-164.9 0-298.6 133.7-298.6 298.6 0 164.9 298.6 554.6 298.6 554.6s298.6-389.7 298.6-554.6c0-164.9-133.7-298.6-298.6-298.6z m0 448a149.3 149.3 0 1 1 0-298.6 149.3 149.3 0 0 1 0 298.6z" fill="${color}" /></svg>`
+    const base64Svg = Buffer.from(svg).toString('base64')
     mapRef.current?.injectJavaScript(`
 if (window.marker) {
     window.marker.remove();
 }
 
-window.marker = L.marker(${JSON.stringify(coordinates)}).addTo(mymap);
+
+
+window.marker = L.marker(${JSON.stringify(coordinates)}, {
```

Comment:
The hitbox of the marker seems to big. When clicking on a room close next to the marker, its does not get selected.